test/fixtures/packageArtifact/artifactForOtherFunction.zip

```diff
@@ -0,0 +1,5 @@
+M Filemode      Length  Date         Time      File
```

Comment:
hmm.. that doesn't look as valid ZIP archive. Or do I miss something?

Reply:
good catch - not sure what happened here - I've ensured it's valid now + changed the naming to be more consistent